mdoc/Test/en.expected-frameworks/FrameworksIndex/One.xml

```diff
@@ -0,0 +1,16 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Framework Name="One">
+  <Type Name="MyFramework.MyNamespace.MyClass">
+    <Member Sig="public double OnlyInClassic { get; set; }" />
```

Comment:
Naming quibble: `Sig` should be explicit about what it is, particularly since `mdoc` supports both C# and IL-based signature generators. I'd suggest `CSharpSignature`.